VAMobile/src/styles/themes/colorSchemes.ts

```diff
@@ -49,6 +49,7 @@ export const lightTheme = {
     linkRow: DSColors.vadsColorGrayWarmLight,
     skeletonLoader: DSColors.vadsColorBaseLighter,
     skeletonLoaderSecondary: DSColors.vadsColorBaseLightest,
+    bullet: colors.grayDark,
```

Comment:
@Sparowhawk I missed this when I approved the PR, but for future color additions/changes can you use colors from the `mobile-tokens` package instead of colors from our `VAColors` file

We're moving away from our custom colors file and exclusively using the colors from `mobile-tokens`. I want to prevent as many merge conflicts as possible with #9199